appium/config/wdio.shared.conf.js

```diff
@@ -16,7 +16,7 @@ exports.config = {
     requires: ['tsconfig-paths/register']
   },
   sync: true,
-  logLevel: 'silent',
+  logLevel: 'debug',
```

Comment:
@tomholub are you ok with this log level? it is pretty verbose but easier to debug issues

Reply:
Surely there could be some options in between silent and debug. Do they have some sort of "normal" verbosity level?

Reply:
debug, info, warn, error

Reply:
Let's try info, see if that gives enough clues without being too verbose